src/index.tsx

```diff
@@ -1,13 +1,6 @@
 import { createRoot } from 'react-dom/client';
-import { HashRouter as Router } from 'react-router-dom';
-
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
+import { Root } from './Root';
```

Comment:
Ensure that the 'Root' component is correctly imported from './Root'. This line seems correct, but double-check the file path and component name to avoid any import issues.